src/focusable-view/FocusableView.d.ts

```diff
@@ -0,0 +1,22 @@
+import { CSSProperties, ComponentClass, ComponentType, FocusEventHandler } from 'react';
+import { Type } from 'tcomb';
+
+export type FocusableViewProps = {
+  children: any, // TODO(typo): t.ReactChildren | (focused: boolean) => JSX.Element
+  onFocus?: FocusEventHandler<any>,
+  onBlur?: FocusEventHandler<any>,
+  tabIndex?: number,
+  component?: ComponentType<any>,
+  ignoreFocus?: boolean,
+  debounce?: number,
+  className?: string,
+  style?: CSSProperties,
+  [k: string]: any
+};
+
+export const Props: {
+  [key: string]: Type<any>
+};
+
+declare const FocusableView: ComponentClass<FocusableViewProps>;
```

Comment:
I've exported all previous components as `export default class Component extends PureComponent<ComponentProps> {}`, so we should define a standard method for this.